lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```
@@ -28,7 +29,7 @@ $-form-child-left-padding: calc(#{$-trigger-side-padding} + #{$-triangle-box-wid
   }
 }
 
-.nx-collapsible-items--expanded {
+.nx-collapsible-items--expanded .nx-collapsible-items__header {
```

We use `>` where it makes sense because it makes the selector slightly more efficient.

```suggestion
.nx-collapsible-items--expanded > .nx-collapsible-items__header {
```